modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PostalAddressResourceTest.java

```diff
@@ -83,6 +91,26 @@ public void setUp() throws Exception {
 			false, false, serviceContext);
 
 		_organization = OrganizationTestUtil.addOrganization();
+
+		ListTypeDefinition subtypeDefinition =
```

Comment:
move the creation of list + entry only in the tests that need them (so 1post and the 2patch tests)